pkg/cluster/k8sres.go

```diff
@@ -7,6 +7,7 @@ import (
 	"path"
 	"sort"
 	"strings"
+	"time"
 
```

Comment:
```suggestion

```

Reply:
Not used. You don't have go.fmt enabled in your IDE?

Reply:
I do, but it runs a little slow sometimes on really large files like this one. Still can't believe I missed this though!

I've now fixed it.

Reply:
It's still there @snarlysodboxer

Reply:
I'm so embarrassed. Fixed for real, for real, this time.